app/assets/stylesheets/application.css

```diff
@@ -57,6 +57,10 @@ and (min-resolution: 192dpi) {
     float: right;
 }
 
+.flash {
```

Comment:
It's better to add bootstrap class like "pull-right" into dev that contains flash messages.